bootstrap.js

```diff
@@ -1973,15 +2024,7 @@ var windowListener = {
 							break;
 						case 'extensions.tabtree.position':
 							let firstVisibleRow = tree.treeBoxObject.getFirstVisibleRow();
-							if (Services.prefs.getIntPref('extensions.tabtree.position') === 1) {
-								browser.appendChild(fullscrToggler);
-								browser.appendChild(splitter);
-								browser.appendChild(sidebar);
-							} else {
-								browser.insertBefore(fullscrToggler, appcontent);
-								browser.insertBefore(sidebar, appcontent);
-								browser.insertBefore(splitter, appcontent);
-							}
+							setTTPos(Services.prefs.getIntPref('extensions.tabtree.position'));
 							tree.view = view;
```

Comment:
What I'm trying to say. If the bookmarks sidebar is hidden and in that moment the tab tree position is changed from "Right" to "In the sidebar (at the top)" then the tab tree looks empty and doesn't work.